### ISSUE TITLE
render in FileSvc missing

### ISSUE THREAD

**Issue by wietze** [mitre/caldera#308]
In commit abdd44bb5ba3b56f911b866a9ff0b301ebf6dc0e, the `/file/render` route and related code was deleted without explaning why. The Adversary plugin relies on this route, [for instance](https://github.com/mitre/adversary/blob/master/static/js/operation.js#L797-L807) to display the PowerShell script to install the CAgent to new machines.

**Comment by ArtificialErmine** [comment]
Yep, changes were made to the back end that had unexpected side effects on adversary mode. I've introduced a pull request (https://github.com/mitre/adversary/pull/47), with fixes for this and some other bugs.
